uwb_platform_interface/lib/uwb_platform_interface.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+///Contains all the calls that needs to be implemented in the platform specific plugins
```

Comment:
There should be a space between the `///` comment keyword and the first character of the description. Also each sentence should end with a period.

```suggestion
/// Contains all the calls that needs to be implemented in the platform specific plugins.
```